### DISCUSSION TITLE
Grove - 125KHz RFID Reader | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1851] [Q&A]
# Grove - 125KHz RFID Reader | Seeed Studio Wiki

This Grove-125KHz RFID Reader is a module used to read uem4100 RFID card information with two output formats: Uart and Wiegand. It has a sensitivity with maximum 7cm sensing distance. There is also the electronic brick version of this module. It can help you with project like internet of thing and access control system.

https://wiki.seeedstudio.com/Grove-125KHz_RFID_Reader/

**Comment by megastary** [comment]
Anyone has any tips how to implement this with Raspberry pi?

**Reply by JGrossholtz** [reply]
I have been able to use this [RFID reader with a Raspberry-Pi](https://openest.io/raspberry-pi/integrate-rfid-reader-raspberry-pi/) over the uart. The only issue I had was the 5v power supply. I wrote an article about it on my [blog](https://openest.io/raspberry-pi/integrate-rfid-reader-raspberry-pi/) and provided Python examples
It works well. I have relatively small tags and I still can read them from 5cms away.
